scripts/justmat/paths.lua

```diff
@@ -485,5 +508,12 @@ function grid_redraw()
       g.led(i, 2, 3)
     end
   end
+  
+  if params:get("show_black_keys") == 2 then
+    for index, t in pairs(black_keys) do
```

Comment:
drive-by comment: since `index` isn't read you might follow the convention and replace it w/ `_`; that is:

```lua
for _, t in pairs(black_keys) do ...
```